src/views/payment-sheet-views/apple-pay-view.js

```diff
@@ -84,6 +84,10 @@ ApplePayView.prototype._showPaymentSheet = function () {
     });
   };
 
+  session.oncancel = function () {
```

Comment:
Interesting choice by Apple to go all lower case for event handlers.

Reply:
Yeah, it looks weird, doesn't it? But it's the pattern the DOM APIs have had from the beginning: https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onclick

We just almost never see it because most people do:

```js
input.addEventListener('click', () => {});
```

instead of 

```js
input.onclick = () => {};
```

Honestly, if Apple provided an `addEventListener` alternative, I'd do it just to not have to deal with how ugly `oncancel` looks :)